en/lesson-08.md

```diff
@@ -158,181 +152,216 @@ by using
 ```
 <!-- {% endraw %} -->
 
-(We'll see `\hspace` [again shortly](lesson-11); you might guess that it adds a
-horizontal space.)
-
-In addition to all the aforementioned preamble content, we can use a few other
-things as well
-
-<!-- don't line wrap this table, markdown seems to not support this -->
+## Adding rules
 
-type | description
----  | :--
-`*{num}{string}` | repeats `string` for `num` times in the preamble. With this you can define multiple identical columns.
-`>{decl}` | this will put `decl` before the contents of every cell in the following column (this is useful, e.g., to set a different font for this column)
-`<{decl}` | this will put `decl` after the contents of each cell in the previous column
+A word of advice prior to introducing rules; lines should be used really
+sparsely in tables, and normally vertical ones look unprofessional. In fact,
+for professional tables you shouldn't use any of the standard lines; instead you
+should get familiar with the facilities of the `booktabs` package, which is why
+it is covered here first &ndash; for the sake of completeness the standard
+lines are shown in the more-info page.
 
-The following example uses an italic font for the first column. Remember our
-example with putting a colon in between the first two columns? How about we also
-append that colon to the first column, so that things don't look as clunky.
+`booktabs` provides four different types of lines. Each of those macros has to
+be used as the first thing in a row or following another rule, else you'll get a
+nasty error. Three rule types and their macros are: `\toprule`, `\midrule`, and
+`\bottomrule`. From their names the intended place of use should be pretty
+clear:
 
 <!-- {% raw %} -->
 ```latex
 \documentclass{article}
-\usepackage{array}
+\usepackage{array, booktabs}
```

Comment:
I'd favour splitting over two lines: it makes life easier when commenting out packages or similar.

Reply:
I only not split it into two because it was done like this elsewhere. I'm fine with splitting it into several, which is what I'd have done in my own document anyways.